pages/registration.page.js

```diff
@@ -59,6 +70,14 @@ class RegistrationPage extends BasePage {
     getRegistrationTextElement() {
         return new TextBox(element(registrationTextLocator), "Check text");
     }
+
+    registrationFormEmailErrorTextElement() {
```

Comment:
`registrationFormEmailErrorTextElement` and `registrationFormPasswordErrorTextElement` used same locator, lets create one method for getting error message